openconfig/ygot
# Handle absolute leafrefs being used as keys for a list.

This PR addresses #489, where a panic was thrown where
an absolute path was used in a leafref that was the key
of a list.

This change adds:

```
* (A) integration_tests/schemaops
  - Add a new integration test that works against an
    arbitrary schema to allow complex test cases to
    be reproduced and part of ongoing integration
    tests.
* (M) generator/.gitignore
  - Add a gitignore to ignore built binaries.
* (M) .gitignore
  - Avoid .vscode folders being created.
* (M) ytypes/list(_test).go
  - Ensure that we handle the absolute leafrefs
    correctly.
```
